lib/bibliothecary/parsers/maven.rb

```diff
@@ -109,6 +113,25 @@ def self.parse_gradle_resolved(file_contents)
         end.compact.uniq {|item| [item[:name], item[:requirement], item[:type]]}
       end
 
+      def self.parse_maven_resolved(file_contents)
+        file_contents
+          .split("\n")
+          .map(&method(:parse_resolved_dep_line))
+          .compact
+          .uniq {|item| [item[:name], item[:requirement], item[:type]]}
+      end
+
+      def self.parse_resolved_dep_line(line)
+        dep_parts = line.strip.split(":")
+        return nil unless dep_parts.length == 5
+        # org.springframework.boot:spring-boot-starter-web:jar:2.0.3.RELEASE:compile[36m -- module spring.boot.starter.web[0;1m [auto][m
+        {
+          name: dep_parts[0, 2].join(":"),
+          requirement: dep_parts[3],
+          type: dep_parts[4].split("--").first.gsub(/\e\[(\d+)m/, '').strip #remove control characters from string
```

Comment:
I think these are ANSI color codes that have shown up in the output. I think I'll need to take a deeper look at this because this could become a problem depending on the environment the maven dependency file was generated in.

Reply:
Added a more comprehensive regex sub.